win32ss/reactx/ntddraw/ddsurf.c

```diff
@@ -107,11 +109,11 @@ NtGdiDdBlt(HANDLE hSurfaceDest,
 
     if (pfnDdBlt == NULL)
     {
-        DPRINT1("Warning: no pfnDdBlt");
+        DPRINT1("Warning: no pfnDdBlt\n");
         return DDHAL_DRIVER_NOTHANDLED;
     }
 
-    DPRINT1("Calling on dxg.sys DdBlt");
+    DPRINT("Calling on dxg.sys pfnDdBlt\n");
```

Comment:
I would recommend to not add pfn here, and in the few other places where you did that, but in this topic my opinion is not very strong.

Reply:
I added a few 'pfn' because it is almost everywhere else.
The other consistency option would be to remove it everywhere (in the strings).

Reply:
Yes, most likely. I would recommend that, but it's okay to leave it for later if you would prefer that. As said: I am no extremist in this topic.